src/wwwroot/template-editor/index.html

```diff
@@ -26,9 +26,11 @@
     </mseditor-loader>
   <script src="//cdn.fromdoppler.com/mseditor/v1.0.0-build515/scripts/lib.min.js"></script>
   <script src="//cdn.fromdoppler.com/mseditor/v1.0.0-build515/scripts/app.min.js"></script>
-  <!-- Inject the environment module dependency in build process for template editor-->
+  <!-- Inject the environment constant RELAY_CONFIG definition dependency in the build process -->
```

Comment:
Do you think that it is possible remove this comment?

Reply:
Yea!